src/mmw/apps/modeling/calcs.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+from django.contrib.gis.geos import GEOSGeometry
+
+from apps.modeling.mapshed.calcs import animal_energy_units
+
+ANIMAL_DISPLAY_NAMES = {
+    'sheep': 'Sheep',
```

Comment:
Instead of recreating the mapping here, prefer reusing existing definitions:

```python
from django.conf import settings


ANIMAL_KEYS = settings.GWLFE_CONFIG['AnimalKeys']
ANIMAL_NAMES = settings.GWLFE_DEFAULTS['AnimalName']

ANIMAL_DISPLAY_NAMES = dict(zip(ANIMAL_KEYS, ANIMAL_NAMES))
```